language/evm/hardhat-examples/.gitignore

```diff
@@ -0,0 +1,9 @@
+node_modules
+.env
+coverage
+coverage.json
+typechain
+
+#Hardhat files
+cache
+artifacts
```

Comment:
I like the way you handled this! I was thinking about checking in the whole hardhat project but was wary of including the node packages and other stuff, and the .gitignore approach seems to address this well.